src/main/scala/uk/gov/nationalarchives/dp/client/ContentClient.scala

```diff
@@ -16,7 +16,7 @@ import java.time.format.DateTimeFormatter
 import java.util.UUID
 
 trait ContentClient[F[_]] {
-  def findExpiredClosedDocuments(secretName: String): F[List[Entity]]
+  def findExpiredClosedDocuments(): F[List[Entity]]
```

Comment:
Could/Should we remove the empty brackets?

Reply:
I had to look this up but apparently not. https://docs.scala-lang.org/style/method-invocation.html#arity-0 You should only remove the brackets when the method has no side effects and this one definitely does.